domainservice/src/main/kotlin/com/example/delivery/service/OrderService.kt

```diff
@@ -82,6 +86,11 @@ class OrderService(
     fun updateOrderStatus(id: String, status: String): Mono<DomainOrder> {
         return orderRepository.updateOrderStatus(id, MongoOrder.Status.valueOf(status))
             .map { it.toDomain() }
+            .flatMap {
+                kafkaUpdateOrderStatusSender.sendOrderUpdateStatus(it.toUpdateOrderStatusResponse())
+                    .doOnError { log.error("Couldn't send message to kafka") }
```

Comment:
which message? imagine you have millions of messages and some of them cause exceptions. how do you plan to match exceptions with messages?